cmake/kokkos_tribits.cmake

```diff
@@ -222,12 +222,15 @@ MACRO(KOKKOS_ADD_TEST_EXECUTABLE ROOT_NAME)
     ""
     "SOURCES"
     ${ARGN})
-  KOKKOS_ADD_EXECUTABLE(${ROOT_NAME}
-    SOURCES ${PARSE_SOURCES}
-    ${PARSE_UNPARSED_ARGUMENTS}
-    TESTONLYLIBS ${KOKKOS_GTEST_LIB}
-  )
-  SET(EXE_NAME ${PACKAGE_NAME}_${ROOT_NAME})
+  # Don't do anything if the user disabled the test
+  IF(NOT ${PACKAGE_NAME}_${ROOT_NAME}_DISABLE)
```

Comment:
That does not look right.  As I understand it, disabling a test means not running it but still building the executable.

Reply:
Yes, I think you are right that TriBITS still compiles the test. The relevant code is https://github.com/TriBITSPub/TriBITS/blob/474375581690f008aa78a55f0fb213a3a53df98a/tribits/core/test_support/TribitsAddTestHelpers.cmake#L603-L627.